src/docs/references/stock/index.md

```diff
@@ -0,0 +1,105 @@
+  for a listing. A positive adjustment represents an increase in
+  available stock, while a negative adjustment represents a decrease.
+  For instance, providers add stock by recording a positive stock
+  adjustment, while sales that happen through are transaction are
+  recorded as negative adjustments. Stock adjustments are immutable.
+- A **stock reservation** represents a reservation or purchase of
+  particular quantity of stock through a transaction.
+
+Providers and operators can manage stock for listings through the
+Marketplace API and the Integration API. Stock adjustments can be
+created either
+[directly](https://www.sharetribe.com/api-reference/marketplace.html#create-stock-adjustment),
+or through a
+["compare and set" operation that sets the total available stock](https://www.sharetribe.com/api-reference/marketplace.html#compare-and-set-total-stock)
+consistently, given an expected current total stock quantity. for a
+listing.
+
+All stock reservations are created through transactions, governed by
+your [transaction process](/background/transaction-process/). An
+appropriately constructed transaction process ensures that reservations
+can only be placed for available quantities of stock.
+
+## Stock reservation states
+
+A stock reservation can be in one of several possible states: `pending`,
+`proposed`, `accepted`, `cancelled` or `declined`. Just as with
+[bookings](/references/availability/#booking-states), stock reservations
+change state only through a corresponding transaction transition, using
+one of the
+[stock reservation actions](/references/transaction-process-actions/#stock-reservations).
+
+All stock reservations in `pending` or `accepted` states count as
+reservation against the listing's available stock. On the other hand,
+stock reservations in the `proposed`, `cancelled` or `declined` states
+do not affect the current available stock.
+
+It is important to note that the stock reservation's effect on stock is
+explicitly recorded as individual stock adjustments as the reservation
+progresses through its states. For example, if a proposed reservation is
+accepted, a new stock adjustment with negative sign (decrease of stock)
+is recorded. Similarly, if an `accepted` stock reservation changes state
+to `cancelled`, a positive stock adjustment (increase of stock) is
+recorded. On the other hand, a new stock adjustment is not recorded when
+the change of states does not require one, as is the case for change
+from `pending` to `accepted` state. All this ensures that the set of
+stock adjustments for a particular listing represents an accurate and
+detailed view of how the listing's available stock has changed over
+time.
+
+The figure below illustrates the possible stock reservation states,
+transitions between the states and the corresponding actions that you
+can use in your transaction process.
+
+![Stock reservation states](./stock-reservation-states.png)
```

Comment:
Since whimsical is private, how should we do that? I guess that Whimsical's security model is good and having a public link won't actually grant access to the board, but what if that changes?